website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+- This pattern indicates that important content and elements should be placed upon the **reading gravity path**
+- This pattern is most prevalent in content with little to no typographic hierarchy (i.e. long-form paragraphs)
+- This pattern is considered "in harmony" with natural reading direction
+
+### Z-shaped pattern
+
+A Z-shaped pattern refers to a pattern that follows the shape of the letter Z:
+
+1. The users eyes start in the top left and move horizontally to the right
+2. Upon reaching the end (or terminal) of the line, the eyes move diagonally to the next line
+3. The readers eyes end at the bottom right of the content
+
+![Visual example of a Z-shaped pattern](/assets/patterns/button-alignment/z-shaped-pattern.png)
+
+#### Characteristics
+
+- There are _technically_ no fallow areas (areas that recieve less attention or lack comprehension) as in the Gutenberg pattern, though this is not always the case in practice.
+- This type of pattern is most prevalent in task-based functions or sequences of elements; i.e. a form or series of onboarding steps.
+- This pattern assumes that users read through _all_ of the content sequentially.
+
+### F-shaped pattern
+
+An F-shaped pattern loosely follows the shape of the letter F, first identified through eye-tracking and heatmap studies by the [Neilsen/Norman Group](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content-discovered/).
+
+![Visual example of an F-shaped pattern](/assets/patterns/button-alignment/f-shaped-pattern.png)
+
+**We probably can't use this image directly on our webiste :(**
+
+![An example of eye tracking and heatmaps from the NN/group](/assets/patterns/button-alignment/f-shaped-pattern-nn-group.jpeg)
+
+#### Characteristics
+
+The F-shaped pattern is a prominent example of how users scan through content and how most users don't read content completely.
+
+- Information at the top will generally have increased comprehension compared to information further down the page.
+    - This is anecdotally true for all reading and layout patterns, but is best captured in a heatmap analysis.
+- Information of lesser importance should be concise, potentially bulletted, and aligned to the left hand side of the page, pattern, or component.
+- This layout assumes users only scan content, rather than reading word-for-word.
+
+### Reading patterns in practice
+
+All three of these reading patterns are relevant and useful when observed in isolation, however, the reality of how users read, consume, and comprehend content is likely a combination of multiple reading patterns that change and overlap depending on the context. With that in mind, understanding these patterns and how they are relevant to different types of UX content will help to produce the best experience.
+
+## Key concepts
+
+- No one layout or reading pattern is perfect, users will generally shift between different patterns depending on the content.
```

Comment:
```suggestion
- No particular layout or reading pattern is perfect, users will generally shift between different patterns depending on the content.
```

Reply:
"one" doesn't seem quite clear here although there are other word choices than I suggested if you think there's a better option.